contrib/export/v034/gov/types.go

```diff
@@ -0,0 +1,126 @@
+package gov
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"time"
+)
+
+type GenesisState struct {
+	StartingProposalID uint64                `json:"starting_proposal_id"`
+	Deposits           []DepositWithMetadata `json:"deposits"`
+	Votes              []VoteWithMetadata    `json:"votes"`
+	Proposals          []Proposal            `json:"proposals"`
+	DepositParams      DepositParams         `json:"deposit_params"`
+	VotingParams       VotingParams          `json:"voting_params"`
+	TallyParams        TallyParams           `json:"tally_params"`
+}
+
+type DepositWithMetadata struct {
+	ProposalID uint64  `json:"proposal_id"`
+	Deposit    Deposit `json:"deposit"`
+}
+
+type VoteWithMetadata struct {
+	ProposalID uint64 `json:"proposal_id"`
+	Vote       Vote   `json:"vote"`
+}
+
+type Proposal struct {
+	Content `json:"content"` // Proposal content interface
+
+	ProposalID       uint64             `json:"proposal_id"`        //  ID of the proposal
+	Status           gov.ProposalStatus `json:"proposal_status"`    // Status of the Proposal {Pending, Active, Passed, Rejected}
```

Comment:
same problem if I remove `gov.` and use the locally copypasted version of it, it panic:

https://github.com/cosmos/cosmos-sdk/pull/4471/files#r293900265